src/plrmodes/reimu_b.c

```diff
@@ -602,7 +602,7 @@ static void reimu_dream_init(Player *plr) {
 
 PlayerMode plrmode_reimu_b = {
 	.name = "Dream Shaper",
-	.description = "Bend the boundaries of space like the ribbon in your hair.",
+	.description = "Utilize techniques you borrowed from an eldritch ’friend,’ and tie the boundaries of spacetime into knots as easily as the ribbons in your hair.",
```

Comment:
Actually, I'm not sure we should directly mention the "borrowing" aspect here at all. Youmu's B shot doesn't mention it either, and Marisa's probably wouldn't too (when/if it's restyled after Patchy's elemental magic). My hope is that the description and the visuals of the shots are just about enough for an observant player to pick up on the idea.

If anything, we could imply that the protags obtained the B-shot powers by studying their "superiors" rather than "borrowing" them, since it lines up better with the "science" theme.

Reply:
Changed!